src/Stuart/Client.php

```diff
@@ -31,6 +32,27 @@ public function validateJob(Job $job)
         }
     }
 
+    public function getSchedulingSlotsAtPickup($city, \DateTime $dateTime)
+    {
+        return $this->getSchedulingSlots($city, 'pickup', $dateTime);
+    }
+
+    public function getSchedulingSlotsAtDropoff($city, \DateTime $dateTime)
+    {
+        return $this->getSchedulingSlots($city, 'dropoff', $dateTime);
+    }
+
+    private function getSchedulingSlots($city, $type, \DateTime $dateTime)
+    {
+        $apiResponse = $this->httpClient->performGet('/v2/jobs/schedules/'.$city.'/'.$type.'/'.$dateTime->format('Y-m-d'));
```

Comment:
add spaces around `.` to match other parts of the code where it concatenates information.